db/seeds.rb

```diff
@@ -23,12 +23,6 @@
   institution_id: institution.id,
   external_account_id: "48883f05-bfe1-46fb-818c-d272ace6a069")
 
-AuthSession.create!(account: account,
```

Comment:
I was thinking that instead of just removing its creation we could leave one in a pending state and another one in a successful state. How about something like this?

```
institution = Institution.create!(name: "local institution",
                                  institution_id: "local_institution_1",
                                  thumbnail_url: "https://picsum.photos/200")

account = Account.create!(name: "Unlinked account DO NOT SYNC",
  budget: budget,
  institution_id: institution.id,
  external_account_id: "48883f05-bfe1-46fb-818c-d272ace6a069")

account_to_be_linked = Account.create!(name: "Account to be linked",
  budget: budget,
  institution_id: institution.id,
  external_account_id: nil)

AuthSession.create!(account: account,
  external_id: "707bbbe7-12e3-41e9-98f9-23277b1fb186",
  status: "success",
  external_account_id: "wadus",
  external_institution_id: institution.institution_id)

AuthSession.create!(account: account_to_be_linked,
  external_id: "707bbbe7-12e3-41e9-98f9-23277b1",
  status: "in_progress",
  external_account_id: nil,
  external_institution_id: "SANDBOXFINANCE_SFIN0000")

```

Reply:
If we do this, then we are back to the square one😅
The reason is this callback:
https://github.com/EduardoSimon/budget.io/blob/main/app/models/auth_session.rb#L4

This calls the GoCardless API and then authentication fails during the seed phase.

Sorry if I'm missing anything, but I'm not able to see why do we need an AuthSession object in the seeds?😅 I don't think they are needed when we already have created accounts with external_account_id. 

If we really need these, then we should create it by skipping the `before_create` callbacks.

Reply:
Nope, you are not missing anything. The thing is that I would like to have at least a ready-to-be-authenticated account when seeding the database for later usage. So how about if we disable the creation hook? BTW I have modified the seeds.rb file as part of #35, but it was just to make all the setup work. Feel free to modify it in this MR, but first please rebase my changes.

Reply:
Sure👍🏻, thanks for the heads-up otherwise I'd have conflicts 😂